FirebaseStorageSwift/Tests/Unit/StorageTests.swift

```diff
@@ -92,10 +100,14 @@ class StorageTests: XCTestCase {
   }
 
   func testStorageNoBucketInConfig() throws {
-    let app = try getApp(bucket: "nil") // Special case code in getApp for "nil".
-    XCTAssertThrowsError(try ExceptionCatcher.catchException {
+    XCTAssertThrowsObjCException {
+      let options = FirebaseOptions(googleAppID: "0:0000000000000:ios:0000000000000000",
+                                    gcmSenderID: "00000000000000000-00000000000-000000000")
+      options.projectID = "myProjectID"
+      let name = "StorageTestsNil"
+      let app = FirebaseApp(instanceWithName: name, options: options)
```

Comment:
Nit: maybe move this outside of the `AssertThrowsObjCException` block to make it clear which part we expect to throw?

Reply:
Done